models/tokenprice/tokenprices.sql

```diff
@@ -0,0 +1,27 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key="block_date||'-'||token_address",
```

Comment:
```suggestion
        unique_key="key",
```
somehow this fails on dbt cloud for incremental builds